neuraxle/base.py

```diff
@@ -41,6 +41,8 @@
 from neuraxle.data_container import DataContainer
 from neuraxle.hyperparams.space import HyperparameterSpace, HyperparameterSamples
 
+RECURSIVE_STEP_SEPARATOR = '__'
```

Comment:
I think a similar constant was already defined somewhere else. In that event, please let's avoid duplicating it.

Reply:
the only constant with '__' in the code base. however, there is places where we don't use the constant (old code). adding constant to old code.